deploy/terraform/README.md

```diff
@@ -0,0 +1,21 @@
+## Requirements
+
+
+The terraform scripts here assume that there is a k8s cluster with at least 3 nodes up and running. The nodes on which mayastor should run should be labeled as usual. Control plane components are to be scheduled on node(s) that have the label `openebs.io/engine=none`.
```

Comment:
Could we use a different label name. Could be useful if we want to run the mayaastor and control plane on the same node locally (to avoid having to deploy more VM's)

Reply:
Yes, we can add what we want, add it to the the variables maybe and have it be based on node names?

Reply:
Ah yes, that would better indeed, then we don't even need to manually label the nodes :+1: